cli/src/test/resources/ide-projects/npm/_ide/urls/node/node/v18.19.1/linux_arm64.urls

```diff
@@ -0,0 +1 @@
+https://nodejs.org/dist/v18.19.1/node-v18.19.1-linux-arm64.tar.gz
```

Comment:
all this stuff in `ide-projects/npm/_ide/urls` can IMHO be removed or why do we add this?